test/e2e/framework/network/utils.go

```diff
@@ -418,8 +421,8 @@ func (config *NetworkingTestConfig) DialFromNode(protocol, targetIP string, targ
 
 		// Check against i+1 so we exit if minTries == maxTries.
 		if eps.Equal(expectedEps) && i+1 >= minTries {
-			framework.Logf("Found all expected endpoints: %+v", eps.List())
-			return
+			framework.Logf("Found all %+v expected endpoints: %+v", len(eps.List()), eps.List())
```

Comment:
added this bc in large ci runs being able to see the number makes it obvious to see that this function, for example, is being called to validate 0 endpoints (imo there's maybe a better way then calling it w/ eps=[] but we can save that debate for later date :)), for now just make the logging clear